src/solvers/elasticity_solver.cpp

```diff
@@ -85,7 +80,7 @@ void ElasticitySolver::CompleteSetup()
   if (m_nat_bdr.size() > 0) {
     for (auto &nat_bc_data : m_nat_bdr) {
```

Comment:
should the "_data" be removed from this variable to be consistent with the rest of the changes?

Reply:
So I've been leaving the `ElasticitySolver` alone for the most part because I expect it to be merged with the `NonlinearSolidSolver` once we figure out the material treatment issue.